goalgorilla/open_social
# Issue #3189003 by mdeny: Expand Social Tagging feature on user profiles

## Problem
As a LU I want to be able to filter on all the attributes added to the profile

## Solution
Expand Social Tagging feature on user profiles

## Issue tracker
https://www.drupal.org/project/social/issues/3189003
https://getopensocial.atlassian.net/browse/YANG-4352

## How to test
- [ ] profile entities as selectable option should be on settings form of social_tagging module
- [ ] tags should appear as filters on search (user tab) automatically
- [ ] selected tags should show on the user profile detail page similar to expertise / interests
- [ ] it should be properly integrated in profile edit form

see: https://www.figma.com/proto/eFn9UYCaSnerZgx9vxsmbR/Open-Social-Design-System?node-id=5097%3A4&viewport=639%2C775%2C0.5&scaling=min-zoom
